pkg/secretless/plugin/sharedobj/internal_plugins.go

```diff
@@ -16,6 +16,7 @@ type InternalPluginLookupFunc func() (plugin.AvailablePlugins, error)
 
 // GetInternalPluginsFunc returns currently available built-in plugins.
 func GetInternalPluginsFunc() (plugin.AvailablePlugins, error) {
+	// New connectors should have an entry in the map below, according to their type (http/tcp)
```

Comment:
we've been trying to standardize on HTTP and TCP when referring to them outside of a command that requires specific syntax.